last-rev-llc/lastrev-libraries
# Fixed an error where invalid contentful queries were leading to faile…

…d webhook update

#### Description

bad queries (for content types that did not exist) were being performed (issue mentioned in slack). but those queries were not being handled correctly, due to an object, not an error being returned.

Added more logging, and handling the error better to allow processing to continue.

---

##### 🔹 Jira Ticket

[IAS-244](https://lastrev.atlassian.net/browse/IAS-244)

##### 🔬 How to test

- change something in contentful, in IAS
- view the webhook response (should be 200)

##### 📸 Screenshots _(if applicable)_

---

##### Changes include
